static/app/components/charts/groupStatusChart.tsx

```diff
@@ -0,0 +1,150 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import MarkLine from 'sentry/components/charts/components/markLine';
+import MiniBarChart from 'sentry/components/charts/miniBarChart';
+import {LazyRender} from 'sentry/components/lazyRender';
+import {t} from 'sentry/locale';
+import type {Group, TimeseriesValue} from 'sentry/types';
+import type {Series} from 'sentry/types/echarts';
+import {formatAbbreviatedNumber} from 'sentry/utils/formatters';
+import theme from 'sentry/utils/theme';
+
+function asChartPoint(point: [number, number]): {name: number | string; value: number} {
+  return {
+    name: point[0] * 1000,
+    value: point[1],
+  };
+}
+
+const EMPTY_STATS: ReadonlyArray<TimeseriesValue> = [];
+
+type Props = {
+  data: Group;
```

Comment:
I completely agree and would take it another step further - the chart component shouldn't even have the responsibility of deciding whether or not to use the filtered or unfiltered stats, especially since it only uses one. The parent component should just decide which one to use and pass the correct stats down to this component.